DESCRIPTION

```diff
@@ -24,7 +24,7 @@ Imports:
     hardhat (>= 1.2.0),
     lifecycle (>= 1.0.3),
     modelenv (>= 0.1.0),
-    parsnip (>= 1.1.0.9001),
+    parsnip (>= 1.1.0.9007),
```

Comment:
```suggestion
    parsnip (>= 1.1.1.9007),
```

though this will conflict with #214, so fine to wait until that's been merged and we merge upstream.

Reply:
good catch!